test/integration-test.py

```diff
@@ -183,6 +197,23 @@ def run_node_test(domain, chall_type, expected_ct_submissions):
 
     verify_ct_submission(expected_ct_submissions, "http://localhost:4500/submissions")
 
+    # Check that the expiration mailer sends a reminder
+    expiry = get_expiry_time(cert_file_pem)
+    no_reminder = expiry + datetime.timedelta(days=-31)
+    first_reminder = expiry + datetime.timedelta(days=-13)
+    last_reminder = expiry + datetime.timedelta(days=-2)
+    urllib2.urlopen("http://localhost:9381/clear", data='')
+    if subprocess.Popen(('cd ../.. ; ' +
+        ('FAKECLOCK=`date -d "%s"` ./bin/expiration-mailer --config test/boulder-config.json ; ' * 3)
```

Comment:
Can this not just be `../../bin/expiration-mailer -config ../../test/boulder-config.json`?